src/features/__tests__/__fixtures__/schemas.ts

```diff
@@ -183,3 +185,63 @@ export const testSchemaConditionalProperties = {
     },
   ],
 } as JSONSchema7;
+
```

Comment:
this fixture file is getting lengthy for my taste, feel free to split just this one into another file if you want, and we can move the rest to match. but it's really not a problem at all